src/main/java/school/lemon/changerequest/java/introduction/hw2/Task3.java

```diff
@@ -3,6 +3,19 @@
 
 public class Task3 {
     public static String comboString(String s1, String s2) {
-        return "";
+
+        if(s1 == null)
+            return "null"+s2+"null";
+        if(s2 == null)
+            return "null"+s1+"null";
+        if (s1.length() == 0)
```

Comment:
You can avoid this additional length checks. Please optimize your solution, so that you'll have only 3 `if` cases and one `else` case